include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_utils.hpp

```diff
@@ -42,6 +42,14 @@ constexpr bool is_GNHWK_GKYXC_GNHWC()
            is_same_v<WeiLayout, tensor_layout::convolution::GKYXC> &&
            is_same_v<OutLayout, tensor_layout::convolution::GNHWK>;
 }
+
+template <typename InLayout, typename WeiLayout, typename OutLayout>
+constexpr bool is_NGKHW_GKYXC_NGCHW()
```

Comment:
There is some kind of confusion with the function name and the tparams order, since the first `NGKHW` layout maybe perceived as `InLayout` whereas it is compared with `OutLayout` - I believe this is because it's designed for conv bwd right? Please at least ad a comment to all those functions that they're for conv bwd.

Reply:
I fixed order of layouts.